src/test/ascii/session.test.ts

```diff
@@ -72,23 +72,88 @@ test('end to end logon', async () => {
 test('session send resendRequest when logged on', async () => {
   const runner: SkeletonRunner = new SkeletonRunner(experiment, 2)
   const factory = experiment.client.config.factory
-  const resend = factory?.resendRequest(1, 2)
+  const resend = factory?.resendRequest(1, 1)
```

Comment:
In this scenario, we've only sent the login message so far, so there's no reason for it to have end seq num = 2 (unless it is 2 on purpose, for some reason)...